internal/provider/resource_packer_channel.go

```diff
@@ -164,26 +139,75 @@ func resourcePackerChannelCreate(ctx context.Context, d *schema.ResourceData, me
 	bucketName := d.Get("bucket_name").(string)
 	channelName := d.Get("name").(string)
 
-	var iteration *packermodels.HashicorpCloudPackerIteration
+	createRestriction := packermodels.NewHashicorpCloudPackerCreateChannelRequestRestriction(packermodels.HashicorpCloudPackerCreateChannelRequestRestrictionRESTRICTIONUNSET)
+	//lint:ignore SA1019 GetOkExists is fine for use with booleans and has defined behavior
+	restrictedRaw, restrictedSet := d.GetOkExists("restricted")
```

Comment:
I see now that unset won't change the restriction. In this case, I think the doc could be clear that set to false is equal to unrestricted. Because you use `otherwise` it sounds like everything different from `restricted = true` will be unrestricted

Reply:
Will do